nilearn/plotting/displays.py

```diff
@@ -539,8 +539,11 @@ def add_contours(self, img, filled=False, **kwargs):
         if filled:
             colors = kwargs['colors']
             levels = kwargs['levels']
-            # Append lower boundary value to '0' for contour fillings
-            levels.append(0.)
+            # contour fillings levels should be given as (lower, upper).
+            # we force append each upper level to inf than zero. Appending zero
+            # will break the contour fillings.
+            # See nilearn issue #931 which has matplotlib reference for this problem.
```

Comment:
I don't think that these comments are necessary since the previous behavior was based on a bug. The first is useful though.